package.json

```diff
@@ -16,5 +16,8 @@
   "license": "ISC",
   "devDependencies": {
     "husky": "^9.0.11"
+  },
```

Comment:
This was installed in the root folder of the project, so should be removed from here where there should be no dependencies, and be installed in the frontend instead.